packages/core/database/src/repairs/index.ts

```diff
@@ -0,0 +1,52 @@
+import type { Database } from '..';
+import { removeOrphanMorphTypes } from './operations/remove-orphan-morph-types';
+import type { Operation, RepairManager } from './types';
+
+export type * from './types';
+
+export const createRepairManager = (db: Database): RepairManager => {
```

Comment:
I'm not sure I see why we need the complexity of doing a generic list of operation when we are still statically importing them at the top of the file 🤔  

Can't this be simpler and avoid all the manual typings ? 

```
import * as operations from './operations';

createRepairManager = () =>  {

  // some internals to the service like stats, loggering, debugging ...

 return {
   removeOprhanMorphTypes:  operation.removeOprhanMorphTypes 
  
   // or just this if you need to wrap. 
   removeOprhanMorphTypes: {
      // some log
			return operations.removeOrphanMorphTypes()
			// some log
   ...
 }
}
```

Reply:
Sounds good. Much simpler now :)